src/codegen/codegen_c_visitor.hpp

```diff
@@ -614,9 +614,11 @@ class CodegenCVisitor: public visitor::ConstAstVisitor {
     /**
      * Determine the variable name for a global variable given its symbol
      * \param symbol The symbol of a variable for which we want to obtain its name
+     * \param use_instance Should the variable be accessed via the (host-only)
+     * global variable or the instance-specific copy (also available on GPU).
```

Comment:
minor / could be a separate decision - as mentioned on slack, I was wondering if 2 copies is a good idea.

Reply:
This has now been changed. Global variables go in a plain data struct `Memb_list::global_variables[_size]` where CoreNEURON handles the host/device movement (as it does for `celsius` etc.). The instance struct now contains no data, only pointers to range variables (inside `ml->data`), true global variables (`celsius`), and mechanism global variables (i.e. `ml->global_variables`), and the translated mechanism .cpp copies this in/out lazily.